pages/glossary/background-colors-of-text.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Background Colors Of Text Nodes
+key: background-colors-of-text
+---
+
+All colors of the pixels, excluding the [foreground colors](#foreground-colors-of-text) pixels, in an [bounding box around text](#bounding-box-around-text) around each [visible](#visible) character in a [text node](https://dom.spec.whatwg.org/#text).
```

Comment:
The colors of all pixels, excluding the [foreground color](#foreground-colors-of-text) pixels, in a [bounding box](#bounding-box-around-text) around each [visible](#visible) character of a [text node](https://dom.spec.whatwg.org/#text).